frontends/web/src/utils/websocket.ts

```diff
@@ -19,10 +20,14 @@ import { qtSubscribePushNotifications } from './qttransport';
 import { androidSubscribePushNotifications } from './androidtransport';
 import { runningInAndroid, runningInQtWebEngine } from './env';
 
-let socket = null;
-const currentListeners = [];
+type SocketInstance = WebSocket | undefined;
```

Comment:
calling a type an instance is very confusing to me. an instance is usually a run time value of a type.

Reply:
Changing to `let socket: WebSocket | undefined;`

Reply:
fixed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/3e9e224a6bdc7f8c435a02e84d945e76e01129c6..79dca499b2c19ed719e18332c610768c5c3adde9